diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 8d1ea713e1dd03f52783bbceea9d85815a358713..d29e8cff37c86196e0f1ef6df2c00bdf70d6323a 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -285,5 +285,6 @@ function expand_(str, max, maxLength, isTop) {
         str = m.post;
     }
     return acc;
 }
+module.exports = Object.assign(expand, exports);
 //# sourceMappingURL=index.js.map
